src/Sql/Sql/ManagedDatabase/Cmdlet/MoveCopy/StopAzureSqlManagedDatabaseMove.cs

```diff
@@ -0,0 +1,43 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Sql.ManagedDatabase.Model;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedDatabase.Cmdlet
+{
+    [Cmdlet(VerbsLifecycle.Stop, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceDatabaseMove",
+        SupportsShouldProcess = true,
+        DefaultParameterSetName = MoveCopyManagedDatabaseByNameParameterSet),
+        OutputType(typeof(bool))]
+    public class StopAzureSqlManagedDatabaseMove : MoveCopyAzureSqlManagedDatabaseBase
+    {
+        protected override string ShouldProcessConfirmationMessage => "Stop ongoing managed database move operation";
+
+        [Parameter(Mandatory = true, ValueFromPipeline = true, ParameterSetName = "CancelManagedDatabaseMoveByMoveModelObject", HelpMessage = "Object that is returned from start move operation.")]
+        [ValidateNotNullOrEmpty]
+        public MoveCopyManagedDatabaseModel MoveModelObject { get; set; }
+
+        protected override MoveCopyManagedDatabaseModel PersistChanges(MoveCopyManagedDatabaseModel model)
+        {
+            if (MoveModelObject != null)
```

Comment:
is this expected to ever be null? (with `Mandatory = true` and `ValidateNotNullOrEmpty`)

Reply:
nvm didn't see the base class 😁